x/crosschain/keeper/msg_server_update_tss.go

```diff
@@ -70,3 +72,9 @@ func (k msgServer) UpdateTssAddress(
 
 	return &types.MsgUpdateTssAddressResponse{}, nil
 }
+
+// GetChainsSupportingMigration returns the chains that support migration.
+func (k *Keeper) GetChainsSupportingMigration(ctx sdk.Context) []chains.Chain {
```

Comment:
```suggestion
func (k *Keeper) GetChainsSupportingTSSMigration(ctx sdk.Context) []chains.Chain {
```
We should explicitely mention TSS migration everywhere as there can be any kind of migration